06_homework_online_store/script.js

```diff
@@ -0,0 +1,116 @@
+let bikes = [{
+    brand: "BMW",
+    model: "R100",
+    engineCapacity: 980,
+    colour: "white",
+    imgSrc: 'images/bmw.jpg',
+    price: 2500,
+    condition: {
+        year: 1993,
+        mileage: 40000
+    },
+    inStock: true,
+    description: "BMW motorcycle series, using a two-cylinder four-stroke boxer engine of 980 cc capacity. " +
+        "Model production began in 1976, with a premature shutdown and subsequent restart in 1985," +
+        " and final completion of the series in 1996."
+},
```

Comment:
indentation :) 
And you don;t need next empty line. So object would go one after another
```
{
  ...
},
{
  ...
}
```

Reply:
Thank you, i need to train it.

Reply:
Fixed.